microsoft-365/admin/activity-reports/onedrive-for-business-usage.md

```diff
@@ -52,7 +52,7 @@ You can get a view into OneDrive for Business usage by looking at the **Accounts
 |6.  <br/> | On the **Accounts** chart, the Y axis is the number of OneDrive accounts.  <br/>  On the **Files** chart, the Y axis is the number of files stored in OneDrive.  <br/>  On the **Storage** chart, the Y axis is the amount of OneDrive storage used.  <br/>  The X axis on all charts is the selected date range for this specific report.  <br/> |
 |7.  <br/> |You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Files** chart, select **Total files** or **Active files**. On the **Accounts** chart, select **Total accounts** or **Active accounts**. Or on the **Storage** chart, select **Storage used**. Changing your selection doesn't change the information in the table.  <br/> |
 |8.  <br/> | The table shows you a breakdown of data for each user's OneDrive. To appear in the table, a user needs to have been assigned a product license that includes OneDrive, and they need to have SharePoint Online turned on. The user also needed to either sign in to the OneDrive sync client, or browse to their OneDrive using a web browser.  <br/>  If the OneDrive has had file activity, it will have the latest date that the file activity was performed. The rows in the table are sorted by **Last activity date** so the OneDrive with the most recent file activity appears at the top of the list.  <br/>  You can add or remove columns from the table.  <br/> ![Column options](../../media/onedriveusage-columns.png)  <br/> **URL** is the web address for the user's OneDrive.  <br/> **Deleted** is the deletion status of the OneDrive. It takes at least 7 days for accounts to be marked as deleted.  <br/> **Owner** is the username of the primary administrator of the OneDrive.  <br/> **Owner principal name** is the email address of the owner of the OneDrive.  <br/> **Last activity date (UTC)** is the latest date a file activity was performed in the OneDrive. If the OneDrive has had no file activity, the value will be blank.  <br/> **Files** is the number of files in the OneDrive.  <br/> **Active files** is the number of active files within the time period.<br/> NOTE: If files were removed during the specified time period for the report, the number of active files shown in the report may be larger than the current number of files in the OneDrive. >  Deleted users will continue to appear in reports for 180 days.<br/>**Storage used (MB)** is the amount of storage the OneDrive uses in MB. <br/>  If your organization's policies prevent you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```

Comment:
```suggestion
|8.  <br/> | The table shows you a breakdown of data for each user's OneDrive. To appear in the table, a user needs to have been assigned a product license that includes OneDrive, and they need to have SharePoint Online turned on. The user also needed to either sign in to the OneDrive sync client, or browse to their OneDrive using a web browser.  <br/>  If the OneDrive has had file activity, it will have the latest date that the file activity was performed. The rows in the table are sorted by **Last activity date** so the OneDrive with the most recent file activity appears at the top of the list.  <br/>  You can add or remove columns from the table.  <br/> ![Column options](../../media/onedriveusage-columns.png)  <br/> **URL** is the web address for the user's OneDrive.  <br/> **Deleted** is the deletion status of the OneDrive. It takes at least 7 days for accounts to be marked as deleted.  <br/> **Owner** is the username of the primary administrator of the OneDrive.  <br/> **Owner principal name** is the email address of the owner of the OneDrive.  <br/> **Last activity date (UTC)** is the latest date a file activity was performed in the OneDrive. If the OneDrive has had no file activity, the value will be blank.  <br/> **Files** is the number of files in the OneDrive.  <br/> **Active files** is the number of active files within the time period.<br/> NOTE: If files were removed during the specified time period for the report, the number of active files shown in the report may be larger than the current number of files in the OneDrive. Deleted users will continue to appear in reports for 180 days.<br/>**Storage used (MB)** is the amount of storage the OneDrive uses in MB. <br/>  If your organization's policies prevent you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```

Reply:
Please remove the > symbol at the beginning of the sentence "Deleted users will continue to appear in reports for 180 days."

Reply:
Ok, I will Johan, but that was already present on the article.

Reply:
Thank you Andres.